flytekit/clients/auth/authenticator.py

```diff
@@ -54,11 +54,13 @@ def __init__(
         header_key: str,
         credentials: Credentials = None,
         http_proxy_url: typing.Optional[str] = None,
+        verify: typing.Optional[typing.Union[bool, str]] = None,
```

Comment:
why is this type a union of a bool and a string?

Reply:
https://requests.readthedocs.io/en/latest/api/#requests.request
http requests is designed like this 
> verify – (optional) Either a boolean, in which case it controls whether we verify the server’s TLS certificate, or a string, in which case it must be a path to a CA bundle to use. Defaults to True.

Reply:
oh i see